static/js/click_handlers.js

```diff
@@ -644,13 +644,13 @@ export function initialize() {
     });
 
     // this will hide the alerts that you click "x" on.
-    $("body").on("click", ".alert-box > div .exit", function () {
+    /*$("body").on("click", ".alert-box > div .exit", function () {
         const $alert = $(this).closest(".alert-box > div");
         $alert.addClass("fade-out");
         setTimeout(() => {
             $alert.removeClass("fade-out show");
         }, 300);
-    });
+    });*/
```

Comment:
I assume the plan is to delete these since you moved them to another file?

Reply:
Yes, perhaps I should force-push another commit with these deleted.